dex/src/main/java/io/crate/data/Projector.java

```diff
@@ -31,4 +31,8 @@
  */
 public interface Projector extends UnaryOperator<BatchIterator> {
 
+    default boolean isScrollable() {
```

Comment:
I think `isScrollable` is a bit confusing as now projectors which have batchIterators which are scrollable have set it to false. I think it should be renamed to something like `cheapScroll` or `providesFreeScroll` (both bad as well, but maybe you've a better idea).